javascript/node/segment.js

```diff
@@ -0,0 +1,38 @@
+// Install the official Segment Node.js SDK: npm i --save analytics-node
+import Analytics from "analytics-node"; // Tested with version 6.2.0
+
+let analytics;
+
+const segmentDestinationInterface = {
+  make: (_env, apiKey) => {
+    analytics = new Analytics(apiKey, { flushAt: 1 });
+  },
+
+  identify: (userId) => analytics.identify({ userId }),
```

Comment:
Do you think we need to null check `analytics`?

Reply:
No we shouldn't have to as identify will always be called after init.